messaging-experiments-82.json

```diff
@@ -321,6 +321,93 @@
     },
     "targeting": "browserSettings.update.channel == 'release' && localeLanguageCode == 'en' && region == 'US' && 'app.shield.optoutstudies.enabled'|preferenceValue"
   },
+  {
+    "id": "WIZARD-LESS-PASSWORD-IMPORT-AUTOCOMPLETE-EXPERIMENT",
+    "enabled": true,
+    "targeting": "firefoxVersion >= '82.' && browserSettings.update.channel != 'release' && localeLanguageCode == 'en' && (isFirstStartup || 'bug-1668315-message-wizard-less-password-import-autocomplete-exper-release-82-83' in activeExperiments)",
+    "arguments": {
+      "slug": "bug-1668315-message-wizard-less-password-import-autocomplete-exper-release-82-83",
```

Comment:
Turns out I hardcoded the expected slug when this originally landed in firefox 82… https://searchfox.org/mozilla-central/rev/40b561b9accbff54387edcba2b8b4a3c3828110e/toolkit/components/passwordmgr/LoginHelper.jsm#68-69